packages/react-composites/src/composites/MeetingComposite/PeopleButton.tsx

```diff
@@ -15,6 +15,7 @@ export const PeopleButton = (props: ControlBarButtonProps): JSX.Element => {
     <ControlBarButton
       {...props}
       strings={strings}
+      labelKey={'peopleButtonLabelKey'}
```

Comment:
Quickly fixed this also:
![image](https://user-images.githubusercontent.com/2684369/133479215-47028ecb-baf1-40ad-baaf-f842003a0726.png)